sagemcom_api/client.py

```diff
@@ -288,6 +288,15 @@ async def login(self):
         else:
             raise UnauthorizedException(data)
 
+    async def logout(self):
+        actions = {
+            "id": 0,
+            "method": "logOut"
+        }
+
+        response = await self.__api_request_async([actions], True)
+        data = self.__get_response(response)
```

Comment:
I guess this `__get_response` is not actually necessary?

Reply:
You can remove it indeed, since we don't do anything with the response. What is the response?

Not sure if we want to throw a bool (True/False depending on the logout result).